### PR TITLE
Add nanoflow event

### PR DESCRIPTION
**Pull request by jseremba**

### CODE REVIEW

File: src/components/BadgeContainer.ts [hunk 1]
```diff
@@ -112,7 +114,9 @@ export default class BadgeContainer extends Component<BadgeContainerProps, Badge
         let errorMessage = "";
         if (props.onClickEvent === "callMicroflow" && !props.microflow) {
             errorMessage = "A 'Microflow' is required for 'Events' 'Call a microflow'";
-        } else if (props.onClickEvent === "showPage" && !props.page) {
+        } else if (props.onClickEvent === "callNanoflow" && !props.nanoflow) {
+            errorMessage = "A 'Nanoflow' is required for 'Events' 'Call a nanoflow'";
+        }else if (props.onClickEvent === "showPage" && !props.page) {
```
Comment (djkwagala) [inline src/components/BadgeContainer.ts]: missing a space between `}` and `else`

File: src/components/BadgeContainer.ts [hunk 2]
```diff
@@ -137,6 +141,15 @@ export default class BadgeContainer extends Component<BadgeContainerProps, Badge
                     guids: [ mxObject.getGuid() ]
                 }
             });
+        } else if (onClickEvent === "callNanoflow" && nanoflow) {
+            window.mx.data.callNanoflow({
+                context,
+                error: (error: Error) => {
+                    mx.ui.error("Error executing nanoflow " + onClickEvent + "callNanoflow" + " : " + error.message);
```
Comment (djkwagala) [inline src/components/BadgeContainer.ts]: Please correct the error message 
> \`Error executing nanoflow ${nanoflow} : ${error.message}\`

**Review by djkwagala — CHANGES_REQUESTED** [review]
Reminder to bump your version too. as well as changing it in package-lock.json or running npm i again.

File: src/components/BadgeContainer.ts (hunk 1 above)
Comment (jseremba) [reply src/components/BadgeContainer.ts]: Added

File: src/components/BadgeContainer.ts (hunk 2 above)
Comment (jseremba) [reply src/components/BadgeContainer.ts]: Done

**Pull request merged**